transfer/streaming/streaming-02-kafka-to-http/kafka-server/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+services:
+  zookeeper:
```

Comment:
instead of using cp-kafka with zookeeper, please take a look to [`kraft`](https://hub.docker.com/r/bashj79/kafka-kraft), it's also used on connector repo for e2e tests